src/components/Cards/Login.vue

```diff
@@ -78,7 +78,10 @@ export default {
           console.log(err.response)
 
           // If the api gave use details of the error, then use them
-          if (err.response.data.errors) {
+          if (err.response !== undefined &&
```

Comment:
Does someone know a shorthand for achieving the same thing? I'm sure it can be simplified somehow